src/sentry/api/serializers/models/event.py

```diff
@@ -366,12 +369,36 @@ def _get_perf_problem(self, attrs):
         converted_problem["issueType"] = get_group_type_by_type_id(issue_type).slug
         return converted_problem
 
+    def _format_breadcrumb_messages(
+        self, event_data: dict[str, Any], event: Event | GroupEvent, user: User
+    ):
+        breadcrumbs = next(
+            filter(lambda entry: entry["type"] == "breadcrumbs", event_data["entries"]), None
```

Comment:
is `entries` guaranteed to exist in `event_data`? Maybe `event_data.get("entries", ())`?